SIL.Scripture/VerseRef.cs

```diff
@@ -1338,6 +1337,27 @@ static void ParseVerseNumber(string vNum, out int number, out string segment)
 			segment = vNum.Substring(j);
 		}
 
+		/// <summary>
+		/// Parses a verse string and gets the leading numeric portion as a number.
```

Comment:
We should have test cases for "1-", "2.3", "5-7", and "6a"

Reply:
Done